jsonapi/marshal.go

```diff
@@ -444,17 +456,3 @@ func marshalStruct(data MarshalIdentifier, information ServerInformation) (*Docu
 
 	return result, nil
 }
-
-func getStructType(data interface{}) string {
-	entityName, ok := data.(EntityNamer)
```

Comment:
I think we need to save this functionality for pluralising and using structure name as json API name

Reply:
restored this functionality